frontend/src/components/Comments/CommentCard.vue

```diff
@@ -56,7 +58,17 @@ export default {
       console.warn('Not yet implemented')
     },
     report() {
-      console.warn('Not yet implemented')
+      const { type, code, id, comment } = this
+      const reason = prompt(`Why should this ${type === 'Answer' ? 'answer' : 'reply'} be removed?`)
+      let promise
+      if (type === 'Answer') {
+        promise = reportAnswer(code, id, comment.id, { reason })
+      } else {
+        promise = reportReply(code, id, comment.id, { reason })
+      }
+      promise
+        .then(() => alert('Thank you. Your report has been submitted.'))
+        .catch((err) => alert(err.message))
```

Comment:
alert the err?

Reply:
yeep - most likely one is 'you've already submitted a report'

Reply:
i call this paradigm 'storeless'